resources/schema.graphql

```diff
@@ -1,392 +1,379 @@
 type AccessToken {
-  accessToken: String!
```

Comment:
Change of the indents are necessary here?
I think it's better to use one type only tabs or only spaces.
As I see most of the files have spaces, could you use spaces here too?

Reply:
This is file that should not be manually touched in general, it's created by scripts.
Looks like the file was explicitly saved in the editor.